metadata-ingestion/src/datahub/emitter/mcp_builder.py

```diff
@@ -129,6 +129,10 @@ class ProjectIdKey(ContainerKey):
     project_id: str
 
 
+class ExperimentKey(ContainerKey):
+    experiment_id: str
```

Comment:
this looks like a breaking change - is that really what we want?

```suggestion
    id: str
```

Reply:
changed back to id